Makefile

```diff
@@ -146,6 +146,8 @@ install:
 	cp $(TOOLS)/gdb/sgx-lkl-gdb $(PREFIX)/bin
 	cp $(TOOLS)/gdb/gdbcommands.py $(PREFIX)/lib/gdb
 	cp $(TOOLS)/gdb/sgx-lkl-gdb.py $(PREFIX)/lib/gdb
+	cp ${TOOLS}/schemas/app-config.schema.json $(PREFIX)/share/schemas
+	cp ${TOOLS}/schemas/host-config.schema.json $(PREFIX)/share/schemas
```

Comment:
Why in source tree, the schemas are under `tools`, while in the installation, the schemas are under the /opt/sgx-lkl?

Reply:
`tools` in the installation is currently only used for the kernel module, and I don't think it belongs there. I think standard installation folders for data is `share/` but let me know if there's a better folder.